spec/pal/features_spec.rb

```diff
@@ -34,7 +34,8 @@
   it 'only adds the feature once' do
     peval(<<-CODE, pal, nil, inventory)
     $t = get_targets('example')[0]
-    $t.set_feature('shell').set_feature('shell')
+    $t.set_feature('shell')
```

Comment:
We document that set_feature returns undef. It was returning a target so chaining worked. https://github.com/puppetlabs/bolt/pull/876/commits/83222672c5e14c9ac6f3003516e75ff826025d8d#r260004950

Reply:
I think cleaning up the docs is less disruptive. We should consider standardizing for the other set/add inventory functions. I'll comment on the appropriate ticket